pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/WorkflowStatusUpdate.java

```diff
@@ -206,11 +218,11 @@ protected static int createParticipantData(String workflow, String status, Strin
         return participantDataId;
     }
 
-    protected static void updateProbandStatus(String workflow, String status, ParticipantData participantData,
-                                              String fieldSettingsFieldType) {
+    protected static boolean updateProbandStatus(String workflow, String status, ParticipantData participantData,
```

Comment:
Quick javadoc for why boolean is returned?  Also, the test does not check the return value.